### DISCUSSION TITLE
WARNING: Host used command 0x1A which is affected by drive geometry.  How do I get rid of this warning?

### DISCUSSION THREAD

**Discussion by bit-image** [ZuluSCSI/ZuluSCSI-firmware#452] [General]
Can anyone tell me how to properly configure my partitions so that this warning message goes away? I have been using the 'Create 2000M HD0.txt' command in my .ini and various other sizes but I cant seem to get around this warning message. Thanks

**Comment by aperezbios** [comment]
@bit-image Are you having an actual problem, or do you just not like that the warning is being issued? It's for informational purposes, and can nearly always be safely ignored. Without knowing the specifics of your machine/operating system, I can't offer further guidance.

**Comment by bit-image** [comment]
Hi. Not having problems, I was just hoping to get this card running optimally. Since the warning indicates  that performance may be affected, I thought I would pose the question here. Thanks
